demos/toolbar/default-toolbar.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDC Toolbar Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <style>
+      .mdc-toolbar-demo {
+        margin: 0px;
+      }
+      .demo-paragraph {
+        margin: 0px;
+        padding: 20px 16px;
+      }
+    </style>
+    <script src="../assets/material-components-web.css.js" charset="utf-8"></script>
+  </head>
+  <body class="mdc-typography mdc-toolbar-demo">
+    <main>
+      <div class="mdc-toolbar">
+        <div class="mdc-toolbar__section mdc-toolbar__section--align-start">
+          <span class="mdc-toolbar__title">Title</span>
+        </div>
+        <div class="mdc-toolbar__section mdc-toolbar__section--align-end">
+          <section class="mdc-theme--dark">
```

Comment:
Two questions about this:
- What should be the implementation if I do want to embed `icon_toggle` in `toolbar`? Not quite sure but seems like I need to manually set `theme-dark` somewhere in order to get the icon in the right color. If I have the question, I wonder if some else will also have that.
- Not sure if having `role="toolbar"` under a section of `toolbar` will be confusing to people.